### DISCUSSION TITLE
Dom文字修改(重新播放按钮下的replay如何修改为"重新播放")

### DISCUSSION THREAD

**Discussion by Sunshine-Anan** [bytedance/xgplayer#825] [Q&A]
**您使用的西瓜播放器版本是多少？ What version of xgplayer are you using?**
2.31.6

**您使用的操作系统和浏览器分别是？ What OS and browser are you using?**

ios 微信浏览器
**如何复现问题？ What did you do?**

**您期望的播放器正常行为是？ What did you expect to happen?**

**实际播放器的表现是？ What actually happened?**

**可填写您所在的公司和相关产品业务，方便我们提供更好的技术支持 You can write your company and product which uses xgplayer,  for helping us provide better technical support.**

**Comment by yiwen03** [comment]
修改player.lang对象对应的REPLAY属性的值

**Comment by Sunshine-Anan** [comment]
可以给个示例吗?

**Comment by Sunshine-Anan** [comment]
改了好像没用,只能把lang: "zh-cn",就变成了中文的"重播"

**Comment by yiwen03** [comment]
不能修改文案，只能通过选取dom，修改对应的文字了

**Comment by hezhongfeng** [comment]
遇到同样的需求了，目前是在实例化完成，即once("ready")，通过 `document.getElementsByTagName("xg-replay-txt")[0].textContent = "重新播放";` ，这样来修改的
